### ISSUE TITLE
Preferred Pool suggestion

### ISSUE THREAD

**Issue by sfalas01** [RainbowMiner/RainbowMiner#2772]
When using multiple pool providers, there may be several pools for the same algorithm.

For example, I primarily want to use Hashcryptos. However, since many algorithms are unavailable, I plan to use ZergPool for everything else. Unfortunately, I can't eliminate the overlaps unless I do it manually. In my opinion, it would be useful to add a preferred pool status or some kind of pool provider ranking system to address this situation when there are many overlapping pools.

Example - Ranking:
1. Hashcryptos
2. ZergPool
3. Zpool

In this case, the system would prioritize pools from Hashcryptos for each algorithm. If an algorithm is not supported on HashCryptos, then it would fall back to the next pool provider. Therefore, an algorithm would be mined on Zpool only if it does not exist in HashCryptos and ZergPool.

I'm not sure how easy this is to implement, but I thought it would be a nice addition.

**Comment by RainbowMiner** [comment]
Yes, that sounds like a very good idea. I will try to add this feature.

**Comment by RainbowMiner** [comment]
Implemented - pool preference is available in the pool configuration. Closing as completed - thanks for the report!
